src/js/utils/is-in-viewport.js

```diff
@@ -1,11 +1,14 @@
 // https://stackoverflow.com/a/7557433
-module.exports = function isElementInViewport (el) {
+function isElementInViewport (el, win=window,
+                              htmlEl=document.documentElement) {
```

Comment:
Good idea, I will switch it to `docEl`... For your second comment, I think that is what I'm doing by using [ES6 default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)?